app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
The function 'go_to_cafe' is defined to return a string, which is consistent with its return statements. Ensure that this aligns with the task requirements.

Reply:
Yes